src/plugins/table/table-designer-menu.js

```diff
@@ -58,7 +66,12 @@ function TableDesignerMenu({ canDeleteColumn, canDeleteRow, cell, dotType, onCel
     createMenuItem(t, DESIGNER_CELL_ACTION.deleteRow, cell, onCellAction, !canDeleteRow, true),
     createMenuItem(t, DESIGNER_CELL_ACTION.insertColumnBefore, cell, onCellAction, false),
     createMenuItem(t, DESIGNER_CELL_ACTION.insertColumnAfter, cell, onCellAction, false),
-    createMenuItem(t, DESIGNER_CELL_ACTION.deleteColumn, cell, !canDeleteColumn, false)
+    createMenuItem(t, DESIGNER_CELL_ACTION.deleteColumn, cell, onCellAction, !canDeleteColumn, true),
+    createMenuItem(t, DESIGNER_CELL_ACTION.connectToRowBefore, cell, onCellAction, cell.isFirstInColumn),
+    createMenuItem(t, DESIGNER_CELL_ACTION.connectToRowAfter, cell, onCellAction, cell.isLastInColumn),
+    createMenuItem(t, DESIGNER_CELL_ACTION.connectToColumnBefore, cell, onCellAction, cell.isFirstInRow),
+    createMenuItem(t, DESIGNER_CELL_ACTION.connectToColumnAfter, cell, onCellAction, cell.isLastInRow),
+    createMenuItem(t, DESIGNER_CELL_ACTION.disconnectCell, cell, onCellAction, !cell.isConnected)
```

Comment:
In such long signature methods called so many times I find it's easy to introduce a call with wrong values.
I'd suggest to either use named arguments instead of positional ones, or always pass all arguments including the default ones. What do you think?

Reply:
Agree, done. I also moved this function into the component, so it can be reduced to 3 arguments.

Reply:
Great, even better! 👍